R/pr.R

```diff
@@ -0,0 +1,137 @@
+pr_create <- function(branch) {
```

Comment:
At first I though `pr_create()` would create a PR. But it really makes sure you are working on a branch, probably off of `master`. Maybe `pr_init()` or `pr_start()`?

Reply:
I also dabbled with `pr_use()` because you can also use it to switch to an existing PR branch.  I like `pr_init()` too.